do/clean

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env bash
+set -eu -o pipefail
+
+function usage()
+{
+  cat <<EOF
+
+Usage:
+ do/${SCRIPT_NAME}
+ do/${SCRIPT_NAME} --force|-f
+ do/${SCRIPT_NAME} --artifacts|-a
+ do/${SCRIPT_NAME} --docker|-d
+ do/${SCRIPT_NAME} --force-artifacts
+ do/${SCRIPT_NAME} --force-docker
+
+ do/${SCRIPT_NAME} --help|-h
+ do/${SCRIPT_NAME} --usage
+
+EOF
+}
+
+function help()
+{
+  usage
+  cat <<EOF
+
+Clean - remove all artifacts resulting from the main build process.
+This script will remove the "build-output" and "test-output" directory.
```

Comment:
We have been making these variables.  See https://github.com/path-robotics/cmake_template/pull/40/files#diff-d13ea887b34380c8c5e2b11cc4fd64a40bde991ff3dccee0a92d0979c7bded12

Reply:
updated